data retrieval/src/main/java/de/tinf15b4/kino/retrieval/scraper/ZkmFilmpalastScraper.java

```diff
@@ -68,7 +65,7 @@ protected GatheringResult gatherData() {
                         String datestr = e.getAttribute("data-original-title");
 
                         // BAD: Hardcode prices
-                        Map<DayOfWeek, Integer> priceMap = new HashMap<>();
+                        Map<DayOfWeek, Integer> priceMap = new EnumMap<>(DayOfWeek.class);
```

Comment:
I dont know but probably yes. I guess enummap has a fixed size as the enum count is already known. And fixed size drastically speeds up things as we know (List <-> Array)